### ISSUE TITLE
I changed nothing but why the outcome was that bad?

### ISSUE THREAD

**Issue by Fluxma** [RicePasteM/Dome-DETR#7]
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=1500 ] = 0.190
 Average Precision  (AP) @[ IoU=0.25      | area=   all | maxDets=1500 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=1500 ] = 0.425
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=1500 ] = 0.139
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=verytiny | maxDets=1500 ] = 0.070
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=  tiny | maxDets=1500 ] = 0.189
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=1500 ] = 0.243
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=1500 ] = 0.252
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.062
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.341
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=1500 ] = 0.401
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=verytiny | maxDets=1500 ] = 0.185
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=  tiny | maxDets=1500 ] = 0.400
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=1500 ] = 0.479
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=1500 ] = 0.476
 Optimal LRP             @[ IoU=0.50      | area=   all | maxDets=1500 ] = 0.840
 Optimal LRP Loc         @[ IoU=0.50      | area=   all | maxDets=1500 ] = 0.276
 Optimal LRP FP          @[ IoU=0.50      | area=   all | maxDets=1500 ] = 0.427
 Optimal LRP FN          @[ IoU=0.50      | area=   all | maxDets=1500 ] = 0.575

**Comment by RicePasteM** [comment]
There is obviously an issue with this result. Our collaborators have conducted accuracy verification on different models of machines, and there have even been cases where training on different models of machines has achieved results that surpass those in the paper. If you need further assistance, please provide some necessary information to help locate the problem more accurately:
- The complete training log
- The complete training configuration file
- The model of the computing card you are using, as well as other detailed environment information.

**Comment by RicePasteM** [comment]
What are the model and quantity of the GPU(s) you are using? And could you please provide the log file with detailed information (like [this](https://huggingface.co/RicePasteM/Dome-DETR/resolve/main/pretrain_ckpts/Dome-M-AITOD-best.log)) for analysis?
